sensor_msgs/Image RGBImage
sensor_msgs/Image DepthImage
Coordinates pos
Coordinates angle